engine/liveupdate/src/liveupdate.cpp

```diff
@@ -479,6 +487,9 @@ namespace dmLiveUpdate
 
     Result StoreResourceAsync(const char* expected_digest, const uint32_t expected_digest_length, const dmResourceArchive::LiveUpdateResource* resource, void (*callback)(bool, void*), void* callback_data)
     {
+        if (!IsLiveupdateEnabled())
```

Comment:
Q: I know that we are mixing one-line conditionals with and without curly brackets, but which is the style to use in the Defold code base? I noticed that you removed curly brackets in the code you touched a bit further down?

Reply:
I think it's depending on context.

I personally think that a scope (curly braces) isn't needed when all we do is return a value.

Assignments may be different.
And often I leave the braces if I for instance have debugged the scope using a printf or similar.